app/templates/components/billing/account.hbs

```diff
@@ -5,29 +5,32 @@
     {{#if this.isEducation}}
       <Billing::Education 
         @newSubscription={{this.newSubscription}}
-        @account={{this.account}} 
-        @plans={{plans}}
+        @account={{this.account}}
+        @moveToNextBillingStep={{this.moveToNextBillingStep}}
+        @moveToPrevBillingStep={{this.moveToPrevBillingStep}}
```

Comment:
In order to highlight that this is just an argument passthrough, I'd suggest to use this syntax:
```js
@moveToNextBillingStep={{@moveToNextBillingStep}}
@moveToPrevBillingStep={{@moveToPrevBillingStep}}
```
`@` here refers to component arguments.

This also applies to other occurrences.

How does it sound to you?